unkeyed/unkey
# feat: add `engines` property to `package.json`

## What does this PR do?

Adds `engines` property to `package.json`



Fixes #1633



## Type of change




## How should this be tested?

- Use an older version of node and try running the project, you should see the minimum version of node required in the terminal. 

## Checklist



### Required


### Appreciated
